src/ioctls/vm.rs

```diff
@@ -1216,6 +1217,40 @@ impl VmFd {
     pub fn run_size(&self) -> usize {
         self.run_size
     }
+
+    /// Wrapper over `KVM_CHECK_EXTENSION`.
+    ///
+    /// Returns 0 if the capability is not available and a positive integer otherwise.
+    fn check_extension_int(&self, c: Cap) -> i32 {
```

Comment:
I reused the code in system.rs.
Now `check_extension()` is the only caller of this function. Defining it is mainly for possible future extension.